src/plots/mapbox/index.js

```diff
@@ -116,41 +105,9 @@ exports.toSVG = function(gd) {
 
         var subplotDiv = d3.select(opts._subplot.div);
 
-        // Append logo if visible
-        var hidden = subplotDiv.select('.mapboxgl-ctrl-logo').node().offsetParent === null;
-        if(!hidden) {
-            var logo = fullLayout._glimages.append('g');
-            logo.attr('transform', strTranslate(size.l + size.w * domain.x[0] + 10, size.t + size.h * (1 - domain.y[0]) - 31));
-            logo.append('path')
-              .attr('d', constants.mapboxLogo.path0)
-              .style({
-                  opacity: 0.9,
-                  fill: '#ffffff',
-                  'enable-background': 'new'
-              });
-
-            logo.append('path')
-              .attr('d', constants.mapboxLogo.path1)
-              .style('opacity', 0.35)
-              .style('enable-background', 'new');
-
-            logo.append('path')
-              .attr('d', constants.mapboxLogo.path2)
-              .style('opacity', 0.35)
-              .style('enable-background', 'new');
-
-            logo.append('polygon')
-              .attr('points', constants.mapboxLogo.polygon)
-              .style({
-                  opacity: 0.9,
-                  fill: '#ffffff',
-                  'enable-background': 'new'
-              });
-        }
-
         // Add attributions
```

Comment:
These attributions are added to comply with mapbox licence in #4069.
Now using `mapLibre` Most users may want to disable the attributions over the maps "by default".
@ndrezn Something to consider in your plan to discuss next week.
Thank you!

Reply:
What is the change caused by adding these attributions?

Reply:
These attributions created a small mapbox logo in the lower left of all the plots. It's possible, but not mandatory to have such a logo overlay displayed at all times with MapLibre, and users would likely rather see their plots instead, so this logo attribution is removed here. You can see the logo in the lower left disappear [here](https://github.com/plotly/plotly.js/pull/7015/files/070941e213a56e23c8cb9682221699db12390f7c#diff-2d447375b78db7583ae4084b41fba11b9eeb0f67bbcd4913fbfb9a27fd662fb2).